examples/common/scripts/vtadmin-up.sh

```diff
@@ -24,11 +28,13 @@ web_dir="${script_dir}/../../../web/vtadmin"
 vtadmin_api_port=14200
 vtadmin_web_port=14201
 
-echo "vtadmin-api http-origin set to \"http://${hostname}:${vtadmin_web_port}\""
+case_insensitive_hostname=$(echo "$hostname" | tr '[:upper:]' '[:lower:]')
```

Comment:
ahh that was the issue?

Reply:
Yes! 😄  CORS is case sensitive and I think mac OS comes with a default hostname that is capitalized!